generated/provider_dependencies.json

```diff
@@ -57,6 +57,7 @@
     "cross-providers-deps": [
       "apache.hive",
       "cncf.kubernetes",
+      "common.compat",
```

Comment:
We should add `apache-airflow-providers-common-compat>=1.1.0` in the provider.yaml. This one is added automatically but we need to bump version in common.compat to 1.1.0 as we have new fetures there.

Reply:
This causes CI error:

```
  ERROR: Cannot install apache-airflow-providers-amazon==8.26.0.dev0 and apache-airflow-providers-common-compat 1.0.0.dev0 (from /dist/apache_airflow_providers_common_compat-1.0.0.dev0-py3-none-any.whl) because these package versions have conflicting dependencies.
  
  The conflict is caused by:
      The user requested apache-airflow-providers-common-compat 1.0.0.dev0 (from /dist/apache_airflow_providers_common_compat-1.0.0.dev0-py3-none-any.whl)
      apache-airflow-providers-amazon 8.26.0.dev0 depends on apache-airflow-providers-common-compat>=1.1.0.dev0
```

Is there a way to specify that the provider depends on the next releasable provider version?

In OL CI we've solved this by bumping version not during release, but actually after - this way, if the last released version is for example 1.5, the version on main is 1.6

Reply:
You need to change version of the common.compat provider to 1.1.0 (which will make it also ready for release)

Reply:
In `provider.yaml`

Reply:
Done.